pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -89,7 +91,9 @@ type LoadBalancerOpts struct {
 }
 
 type BlockStorageOpts struct {
-	TrustDevicePath bool `gcfg:"trust-device-path"` // See Issue #33128
+	AllowAutoSupported bool   `gcfg:"auto-allow-supported"` // Allow autodetection to fall back to SUPPORTED api versions
```

Comment:
I would vote against this due to it being too fine-grained. Either just check for "CURRENT" or automatically fallback to "SUPPORTED". If users are clamoring for this feature, it could be added later in a backwards-compatible way.